zproject_java.gsl

```diff
@@ -899,6 +948,14 @@ $(project.GENERATED_WARNING_HEADER:)
 #include "$(project.header:)"
 #include "$(my.cname:).h"
 
+// TODO: This doesn't really belong in here
```

Comment:
I'm just fuzzy about a) where would be better and b) the best way to make that happen